tests/nix-channel.sh

```diff
@@ -2,15 +2,15 @@ source common.sh
 
 clearProfiles
 
-rm -f $TEST_HOME/.nix-channels $TEST_HOME/.nix-profile
```

Comment:
Is there a reason why this logic didn't remove `.nix-defexpr` previously?

Reply:
Technically it doesn't need to, but there's no hurt in removing it.